Project.toml

```diff
@@ -29,6 +29,7 @@ julia = "0.7, 1"
 
 [extras]
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+GraphPlot = "a2cc645c-3eea-5389-862e-a155d0052231"
```

Comment:
Should GraphPlot be in deps or extras? 
Same with Neo4j. 
If we don't want it to be a dependency and use it with @require it makes sense not to add it in deps?

Reply:
Yep!